config/config.go

```diff
@@ -58,32 +58,34 @@ type user struct {
 }
 
 type conf struct {
-	HttpListen string            `yaml:"http-listen" env:"OPENSTACK_NETWORK_EXPORTER_HTTP_LISTEN"`
-	HttpPath   string            `yaml:"http-path" env:"OPENSTACK_NETWORK_EXPORTER_HTTP_PATH"`
-	TlsCert    string            `yaml:"tls-cert" env:"OPENSTACK_NETWORK_EXPORTER_TLS_CERT"`
-	TlsKey     string            `yaml:"tls-key" env:"OPENSTACK_NETWORK_EXPORTER_TLS_KEY"`
-	AuthUsers  []user            `yaml:"auth-users"`
-	users      map[string]string `yaml:"-"`
-	OvsRundir  string            `yaml:"ovs-rundir" env:"OPENSTACK_NETWORK_EXPORTER_OVS_RUNDIR"`
-	OvnRundir  string            `yaml:"ovn-rundir" env:"OPENSTACK_NETWORK_EXPORTER_OVN_RUNDIR"`
-	OvsProcdir string            `yaml:"ovs-procdir" env:"OPENSTACK_NETWORK_EXPORTER_OVS_PROCDIR"`
-	LogLevel   string            `yaml:"log-level" env:"OPENSTACK_NETWORK_EXPORTER_LOG_LEVEL"`
-	logLevel   syslog.Priority   `yaml:"-"`
-	Collectors []string          `yaml:"collectors"`
-	MetricSets []string          `yaml:"metric-sets"`
-	metricSets MetricSet         `yaml:"-"`
-	IntBrdNam  string            `yaml:"br-int-name" env:"OPENSTACK_NETWORK_EXPORTER_BR_INT_NAME"`
+	HttpListen        string            `yaml:"http-listen" env:"OPENSTACK_NETWORK_EXPORTER_HTTP_LISTEN"`
+	HttpPath          string            `yaml:"http-path" env:"OPENSTACK_NETWORK_EXPORTER_HTTP_PATH"`
+	TlsCert           string            `yaml:"tls-cert" env:"OPENSTACK_NETWORK_EXPORTER_TLS_CERT"`
+	TlsKey            string            `yaml:"tls-key" env:"OPENSTACK_NETWORK_EXPORTER_TLS_KEY"`
+	AuthUsers         []user            `yaml:"auth-users"`
+	users             map[string]string `yaml:"-"`
+	OvsRundir         string            `yaml:"ovs-rundir" env:"OPENSTACK_NETWORK_EXPORTER_OVS_RUNDIR"`
+	OvnRundir         string            `yaml:"ovn-rundir" env:"OPENSTACK_NETWORK_EXPORTER_OVN_RUNDIR"`
+	OvsDbServerRundir string            `yaml:"ovs-db-server-rundir" env:"OPENSTACK_NETWORK_EXPORTER_OVS_DB_SERVER_RUNDIR"`
```

Comment:
Can you update `etc/openstack-network-exporter.yaml` with the new setting?

Reply:
I update it based on your response to the conversation below

Reply:
Done. Thanks for the reviews. It is highly appreciated!